packages/ti_abusech/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.0.1"
+  changes:
+    - description: Adds a fix to the image not being shown
```

Nit

```suggestion
    - description: Fix invisible package icon
```

Or something similar. User reading this may not understand what "the image" is.